episodes/season_theatrical.yml

```diff
@@ -78,3 +78,36 @@ episodes:
   simpsonsworld_id:
   good: maybe
   guest_stars:
+- title: When Billie Met Lisa
+  season: 99 
+  episode: 6 
+  release_date: 2022-04-22 
+  directors: David Silverman 
+  writers: ["Elisabeth Kiernan Averick", "Broti Gupta", "Al Jean","Cesar Mazariegos", "David Mirkin"] 
+  description: 
+  disneyplus_id: 
+  simpsonsworld_id: 
+  good: maybe 
+  guest_stars: ["Billie Eilish", "Finneas O'Connell"] 
+ - title: Welcome to the Club 
```

Comment:
extra space at the start here! yaml is a stickler about formatting consistency, unfortunately